osu.Game/Screens/Menu/MainMenu.cs

```diff
@@ -104,13 +104,10 @@ private void preloadSongSelect()
 
         public void LoadToSolo() => Schedule(onSolo);
 
-        private void onSolo() => this.Push(consumeSongSelect());
-
-        private Screen consumeSongSelect()
+        private void onSolo()
         {
-            var s = songSelect;
+            this.Push(songSelect);
```

Comment:
This is not the same logic.  With your change, `songSelect` is nulled *after* the call to `Push`, whereas it was previously before it.

Reply:
Why is nulling after a problem?

Reply:
Does it matter?